src/source/Include_i.h

```diff
@@ -65,6 +63,7 @@ extern "C" {
 #include <arpa/inet.h>
 #include <fcntl.h>
 #include <netinet/tcp.h>
+#include <poll.h>
```

Comment:
Because this was earlier satisfied from `libwebsockets.h`

Reply:
To make this platform independent, I think we should pass it as a compiler flag and only include if true.

cmake:
```cmake
include(CheckIncludeFile)
check_include_file("poll.h" HAVE_POLL_H)

if(HAVE_POLL_H)
    add_definitions(-DHAVE_POLL_H=1)
endif()
```

C:
```c
#ifdef HAVE_POLL_H
#include <poll.h>
#endif
```

Reply:
Maybe similar for `ifaddrs.h`